tests/tdscf-2/input.dat

```diff
@@ -0,0 +1,38 @@
+UHF_TDA_cc_pvdz = [0.24880026306449304,
+                   0.28968755925744966,
+                   0.32054964027744337,
+                   0.35741288565867185,
+                   0.39502214228627547,
+                   0.41144173957102564,
+                   0.4445528791268893,
+                   0.4535932124573471,
+                   0.48482278670215617]
+
+molecule ch2 {
+0 3
+
+C           0.000000    0.000000    0.159693
+H          -0.000000    0.895527   -0.479080
+H          -0.000000   -0.895527   -0.479080
+no_reorient
+no_com
+symmetry c1
```

Comment:
are the symm c1 and no_* required? should integrate into driver rather than demanding from user.

Reply:
Jeff noticed that the solver chokes with symmetry, though it shouldn't. I'm looking at it and hope to lift the restriction soon.

Reply:
no_* isn't required, initially took it from the pytest but it passes without. I'll remove them.